index.html

```diff
@@ -439,12 +438,14 @@ <h1>About Nextcloud</h1>
 				<div class="footer-nav">
 					<h1>Resources</h1>
 					<ul>
+						<li><a class="reference external" href="https://nextcloud.com/search/">Search our website</a></li>
 						<li><a class="reference external" href="https://nextcloud.com/install">Download</a></li>
-						<li><a class="reference external" href="https://apps.nextcloud.com">App Store</a></li>
```

Comment:
Hello @MariusBluem ,

why did you remove the links here?

Reply:
yeah, I wouldn't.